controller/work_item_events_test.go

```diff
@@ -186,18 +187,25 @@ func (s *TestEvent) TestListEvent() {
 	})
 
 	s.T().Run("event list ok - description", func(t *testing.T) {
-		fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
+		fxt := tf.NewTestFixture(t, s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
 		svc := testsupport.ServiceAsSpaceUser("Event-Service", *fxt.Identities[0], &TestSpaceAuthzService{*fxt.Identities[0], ""})
 		EventCtrl := NewEventsController(svc, s.GormDB, s.Configuration)
 		workitemCtrl := NewWorkitemController(svc, s.GormDB, s.Configuration)
 		spaceSelfURL := rest.AbsoluteURL(&http.Request{Host: "api.service.domain.org"}, app.SpaceHref(fxt.Spaces[0].ID.String()))
+
+		modifiedDescription := "# Description is modified1"
+		modifiedRenderedDescription := "<h1>Description is modified1</h1>\n"
+		modifiedMarkup := rendering.SystemMarkupMarkdown
```

Comment:
We don't need this. The rendering method would set this.

Reply:
It doesn't set it properly. If I remove the markup from the update payload, the outcome is PlainText. But you're right that the rendered version is set correctly. See 0267189